### DISCUSSION TITLE
Cannot find module '@babel/preset-plugin-syntax-import-meta'

### DISCUSSION THREAD

**Discussion by a6965921** [babel/babel#14553] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1198840/168458667-2c497434-4781-4054-8fdc-516b20580eb1.png)

**Comment by nicolo-ribaudo** [comment, marked as answer]
You have `@babel/plugin-syntax-import-meta` in the `presets` section of your Babel config; it should be in `plugins`.
